src/Components/ADT3DSceneBuilder/Internal/Behaviors/Widgets/WidgetBuilders/DataHistoryWidgetBuilder/DataHistoryWidgetBuilder.tsx

```diff
@@ -183,6 +176,8 @@ const DataHistoryWidgetBuilder: React.FC<IDataHistoryWidgetBuilderProps> = ({
         [formData]
     );
 
+    const theme = useTheme();
+    const customStyles = getWidgetFormStyles(theme);
```

Comment:
can we use our styled pattern instead of this older way of doing styles?

Reply:
changing it now

Reply:
hey @msnyder-msft after checking this, all these styles are coming from "shared" stylings of WidgetForm which is shared by all the forms in widget builder components (e.g. widgetFormContents from WidgetForm.styles). I tried to make it separate for only one classname which seems to me a component specific but I don't see much contribution of it: https://github.com/microsoft/iot-cardboard-js/pull/716/commits/edb3e45e1656bd5af5dba4b6809075ffd171aee1 Is this what you want to see here? If not, do we have an example to consume shared stylings in a component which using the new styling somewhere?

Reply:
nah. If it's shared they are kinda all bolted together. I hadn't noticed that it was shared, I saw `CustomStyles` and assumed it was specific to this component. It's fine to just share the old way if that's waht we have going in these forms. Eventually we can transition all of them at the same time.
I like what you did in that commit, teh only change I would make is to use the `styles` prop instead of classNames. To do that though, you'd need to use `subComponentStyles` and have a property that is of type `IStackStyles`.